src/Analyzer.Core.BuiltInRuleTests/Tests/SQL_Retention_Days.json

```diff
@@ -0,0 +1,22 @@
+﻿[
+  {
+    "Template": "SQLServerAuditingSettings",
+    "ReportedFailures": [
+      // The resource between lines 10 and 33 passes this check because all the elements of the AnyOf are true
+      // The ones between lines 34 and 113 passes because at least one of the elements of the AnyOf is true
+      // And the ones between lines 135 and 173 passes because of empty scopes
```

Comment:
Can you explain the last scenario a bit more?  The idea of "empty scopes" is more of a development concept, and wouldn't make as much sense to someone just creating rules (and reviewing existing tests).  From a rule author perspective, the resources pass because their property values exclude them from the rule criteria.  (You could maybe say the resources are "out of scope" for X/Y reasons, but "scope" in that phrase would have a slightly different meaning and would therefore make sense to a rule author.)